cmd/containerboot/main.go

```diff
@@ -731,16 +731,16 @@ func tailscaledConfigFilePath() string {
 		}
 		cv, err := kubeutils.CapVerFromFileName(e.Name())
 		if err != nil {
-			log.Printf("skipping file %q in tailscaled config directory %q: %v", e.Name(), dir, err)
```

Comment:
This print statement was generating lots of log spam as it was going through all the symlinks and stuff that kube puts into mounted volumes